db/migration/1688372371221-AddPresentationToVariables.ts

```diff
@@ -0,0 +1,103 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class AddPresentationToVariables1688372371221
+    implements MigrationInterface
+{
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `-- sql
+            ALTER TABLE variables
+                ADD COLUMN schemaVersion INT NOT NULL DEFAULT 1,
+                ADD COLUMN processingLevel VARCHAR(30),
+                ADD COLUMN processingLog JSON,
+                ADD COLUMN titlePublic VARCHAR(512),
+                ADD COLUMN titleVariant VARCHAR(255),
+                ADD COLUMN producerShort VARCHAR(512),
+                ADD COLUMN citationInline TEXT,
+                ADD COLUMN descriptionShort TEXT,
+                ADD COLUMN descriptionFromProducer TEXT,
+                -- ADD COLUMN topicTagsLinks JSON,
```

Comment:
I see we're creating `tags_variables_topic_tags` instead of this. Does this mean the python model needs to change? Or should we maybe add a comment here explaining the discrepancy?

Reply:
I removed this outdated line. You're right there's a python ORM model, I'm trying to keep it in sync with these changes.